beacon-chain/sync/service.go

```diff
@@ -301,3 +302,7 @@ type Checker interface {
 	Status() error
 	Resync() error
 }
+
+type EqChecker interface {
```

Comment:
I would call it EquivocationChecker and add a thorough comment. Eq sounds like equals in other programming languages